x-pack/plugins/infra/public/pages/metrics/hosts/components/kpi_charts/kpi_chart.tsx

```diff
@@ -122,4 +122,10 @@ const KPIChartStyled = styled(Chart)`
   .echMetric {
     border-radius: ${(p) => p.theme.eui.euiBorderRadius};
   }
+  button {
+    cursor: default;
```

Comment:
When is this used? I tried without it and I don't see a difference but maybe I am missing something 🤔  Which element do you want to have a default cursor?

Reply:
this is to prevent the cursor from changing to  a hand cursor.
I guess if we already have **pointer-events: none;** it does the same effect so we can remove this.